aau-giraf/web-api
# Backend for weekplanner571

# Description

updated the web-api to support changes made to the api-client and weekplanner, changing the variable nrOfDaysToDisplay to 2 versions based on orientation, also adding a boolean value for each orientation, by updating the database with the correct values.


related to these pull requests: https://github.com/aau-giraf/weekplanner/pull/890/ and https://github.com/aau-giraf/api_client/pull/119

## Type of change


# How Has This Been Tested?

This should be tested together with the weekplanner in this way:
- Get these changes to the web-api locally, then set up the database locally with these changes.
- change line 16 in pubspec.yaml in the weekplanner from `develop ` to `settingsForWeekplanner571`
- run the database
- start the weekplanner
- login and choose a weekplan
- Now when switching to citizen mode it should show 1 day if the table is in portait mode, and 7 in landscape mode (these values can also be changed in the settings menu)


**Development Configuration**
*Type "flutter --version" and "dart --version" in your CMD to check versions.*

* Flutter version:2.0.5
* Dart version:2.12.3

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas, if necessary
- [ ] I have added tests that prove my fix is effective or that my feature works, if necessary
- [ ] I have Acceptance Tested this on an iOS device
- [ ] I have Acceptance Tested this on an Android device
